src/Integration/Laravel/Queue/tests/baseline.json

```diff
@@ -0,0 +1,12 @@
+[
+    {
+        "location": "AsyncAws\\Illuminate\\Queue\\Tests\\Unit\\AsyncAwsSqsQueueTest::testPush",
+        "message": "Ramsey\\Uuid\\Uuid implements the Serializable interface, which is deprecated. Implement __serialize() and __unserialize() instead (or in addition, if support for old PHP versions is necessary)",
```

Comment:
Hm... I wonder if we should really ignore this deprecation...
This is a true deprecation that should/will be fixed.  But this is an indirect deprecation (triggered by a 3rd party).

If we ignore this deprecation, then we'll better have to ignore all the indirect deprecation and only use `max[self]=0` instead of `max[total]=0`

TL;DR = should we care about indirect deprecations or just ignoring them?

Reply:
I think we should just ignore all indirect deprecations.